test/verify/check-no-retry

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+
+
+@noretry
+@skipImage("Do not test BaseOS packages", "rhel-8-3-distropkg")
```

Comment:
This doesn't make much sense in this context? Or do you want to validate that multiple decorators combine properly? Then I'd suggest doing it to one test class only, and changing the comment accordingly.

Reply:
oh nope, I just saw it everywhere on check-example too so I thought it was a thing .-.